pages/likes/index.js

```diff
@@ -43,9 +45,18 @@ export default function LikedCardsDeck({
       <Main>
         <h3>Your Likes</h3>
         <StyledText>
-          Oh no, You have no favorites yet,{" "}
-          <Lynk href="/">browse the games to find some.</Lynk>
+          Oh no, You have no favorites yet,
+          <br /> <Lynk href="/">browse the games to find some.</Lynk>
```

Comment:
`Lynk` has a nice touch! Something of a _Link_ that is _styled_, a Lynk! Cute!

Reply:
Many thanks for the review :))